src/libs/ajax/Metrics.ts

```diff
@@ -24,6 +24,10 @@ export const Metrics = (signal?: AbortSignal) => {
       }));
     }
 
+    // Send event to Appcues and refresh Appcues state
+    window.Appcues?.track(event, details);
```

Comment:
I'm skeptical that we're permitted to this.

We have a service (Bard) in between Terra UI and MixPanel to avoid using MixPanel's JS SDK and sending some information to MixPanel.

Reply:
In regards to compliance, is sending along the `details` information the possible issue, or the fact that we are using the SDK?